### PR TITLE
Support ip in lb ports

### PR DESCRIPTION
**Pull request by alena1108**
@cjellick could you review?

### CODE REVIEW

**Comment by ibuildthecloud** [comment]
No way... It can't be that easy.

**Comment by vincent99** [comment]
Indeed, no way this addresses all the bits needed.

**Comment by alena1108** [comment]
@vincent99 @ibuildthecloud I will work on fixing haproxy config part, changed to WIP

**Comment by alena1108** [comment]
@vincent99 @ibuildthecloud fixed binding on haproxy; leaving as WIP till we decide on hostname routing rules support

File: resources/content/config-content/haproxy/content/etc/haproxy/haproxy.cfg.ftl [hunk 1]
```diff
@@ -47,7 +47,7 @@ defaults
 <#if listener.sourceProtocol == "https"><#assign protocol="http"></#if>
 <#if listener.sourceProtocol == "ssl"><#assign protocol="tcp"></#if>
 frontend ${listener.uuid}_frontend
-        bind *:${sourcePort}<#if (listener.sourceProtocol == "https" || listener.sourceProtocol == "ssl") && certs?has_content> ssl crt /etc/haproxy/certs/<#if !defaultCert??> strict-sni</#if></#if>
+        bind ${listener.sourceIp}:${sourcePort}<#if (listener.sourceProtocol == "https" || listener.sourceProtocol == "ssl") && certs?has_content> ssl crt /etc/haproxy/certs/<#if !defaultCert??> strict-sni</#if></#if>
```
Comment (ibuildthecloud) [inline resources/content/config-content/haproxy/content/etc/haproxy/haproxy.cfg.ftl]: I don't understand this change.  How can haproxy bind to an IP?

**Comment by alena1108** [comment]
@vincent99 @ibuildthecloud ok, final resolution. Binding to specific host ip address on haproxy would work only in host networking mode. As our haproxy runs on the managed, this can't be done. I'm switching back to my original fix, where we just configure port on the host by binding it to specific ip. Then all the traffic coming to this ip/port, will get forwarded to the listening port of haproxy container.

**Pull request merged**